google_drive/templates/html/google_calendar_edit.html

```diff
@@ -0,0 +1,68 @@
+{% load i18n %}
```

Comment:
Do we need to perform an accessibility review for any HTML changes we make?  I know that @cptvitamin usually wants to see "aria" attributes added to various elements, but I'm not sure when they are actually required.